core/src/rpc_pubsub.rs

```diff
@@ -235,6 +257,10 @@ impl RpcSolPubSub for RpcSolPubSubImpl {
         pubkey_str: String,
         config: Option<RpcProgramAccountsConfig>,
     ) {
+        if let Err(err) = self.check_subscription_count() {
+            subscriber.reject(err).unwrap();
```

Comment:
All these `.unwrap()`s are infallible?

Reply:
They should be, but doesn't hurt to be extra-careful.